### DISCUSSION TITLE
iOS app Notifications Don't Work

### DISCUSSION THREAD

**Discussion by daBee** [community/community#106592] [Mobile]
### Select Topic Area

Bug

### Body

I can't get notifications to work for the Github iOS app.  Tried both the iOS and the app settings.  

Any ideas how to get this to work?  I've turned everything on and still not getting anything.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @daBee ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by harundemirkaya** [comment]
The best way to send notifications in mobile apps is with Firebase Cloud Messaging, which is completely free. Create a firebase project for your project from Firebase Console. Then add the necessary settings for your mobile app (package name, etc.) Then go to your project settings via XCode and enable Background Modes and Push Notification in the Add Capabilities section. After enabling Background Modes, don't forget to check background fetch and remote notifications from the options. Then sign in to your Apple Developer account. In the Identifier and certificates section, while the push notification service of your application is enabled, create a certificate for this service. Install this certificate you created from the fcm settings for your application from firebase settings. In this way, your notifications will now work.

**Reply by daBee** [reply]
I'm talking about the Github iOS app.  It isn't/wasn't telling me about any replies here on Github.

**Reply by harundemirkaya** [reply]
😂😂 Sorry, man, I thought you were asking about development. I don't know what you're wondering about.

**Reply by giorgosph** [reply]
@harundemirkaya 
I am building my application using Expo (EAS). I have created an Apple Developer Account and created a build for internal distribution to test it on an iPhone. I have difficulties setting up push notifications as I am not using Xcode. If you can help I have created a new discussion for that:

https://github.com/orgs/community/discussions/126577

Thank you

**Comment by daBee** [comment]
Seems to be working somewhat.  I'll close this.
